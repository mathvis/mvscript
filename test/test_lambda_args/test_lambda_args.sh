#!/bin/bash
helix test_lambda_args.mvs test_lambda_args.out --vsplit
