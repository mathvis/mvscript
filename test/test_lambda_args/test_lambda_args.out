Expr (LambdaFunc [(VarIdentifier "a",int)] (Just (Block (FunctionBlock void) [Expr (VarIdentifier "a")])))
